client/src/redux/store.js

```diff
@@ -0,0 +1,5 @@
+import { createStore, applyMiddleware } from 'redux';
+import { rootReducer } from './reducers';
+const thunk = require('redux-thunk').default;
```

Comment:
use `import` so it is consistent with other style of modules

Reply:
This was also weird, thunk didn't work when I imported the same way as the others. 
I can try again, but this was a stackoverflow solution when people had the same issue